pages/api/users/index.ts

```diff
@@ -19,9 +23,13 @@ export default async function handlePrisma(
   res: NextApiResponse,
 ) {
   try {
+    const validationRequest = await validateRequest(req, res);
+    if (!validationRequest.session || !validationRequest.user) {
+      throw new HttpError('You are unauthorized!', 401);
+    }
```

Comment:
Ehdottaisin, että loisit `backend/auth.ts` tiedostoon uuden `requireLogin(req, res)` funktion, joka kutsuisi `validateRequest` funktiota ja tekisi tässä olevan tarkistuksen että sieltä palautu käyttäjä. Näin ei tarvitse tehdä tätä tarkistusta joka paikkaan missä pitää olla kirjautunut käyttäjä.

Reply:
Hyvä idea! Paljon parempi tapa! :+1: